packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    v-if="banner"
+    class="relative flex items-center px-4 py-2 transition text-tiny group"
+    :class="getBannerColor(banner.type)"
+  >
+    <icon-lucide-info class="mr-2 text-white" />
+    <span class="text-secondaryDark">
+      <span v-if="banner.alternateText" class="md:hidden">
+        {{ banner.alternateText(t) }}
+      </span>
+      <span class="<md:hidden">
+        {{ banner.text(t) }}
```

Comment:
As far as the banner is concerned, it does not need to know about the i18n, its job is to take the text and display it, it is who ever is using the banner, needs to care about whether the text should be internationalized.

i have suggested changes on the banner.service.ts also. let me know what you think. 

```suggestion
        {{ banner.text }}
```